deps/rabbit/src/rabbit_channel.erl

```diff
@@ -729,6 +731,10 @@ handle_info({{'DOWN', QName}, _MRef, process, QPid, Reason},
             handle_eol(QRef, State)
     end;
 
+handle_info({'DOWN', _MRef, process, Pid, normal}, State) ->
+    ?LOG_DEBUG("Process ~0p monitored by channel ~0p exited", [Pid, self()]),
```

Comment:
As mentioned in chat this should include `process_info` information, as well as `process_info(Pid, current_stacktrace)` to figure out where the message is coming from.

Reply:
The reason is also `normal`, so it didn’t crash.

Reply:
No you're right if we have the DOWN message then we most likely won't get anything out of it. What would help is a test run that has tracing enabled but only for the spawn events, this way we have the MFA of all processes that start, and we can see which process it was that DOWN when we reproduce.

Reply:
I never played with tracing. How should I enable it for the entire set of testsuites?